ocamltest/ocamltest_stdlib.ml

```diff
@@ -173,6 +173,20 @@ module Sys = struct
     with_output_file ~bin:true dest @@ fun oc ->
     copy_chan ic oc
 
+  let rec copy_directory src dst =
```

Comment:
This implementation can be tripped up by symlinks - I don't think it matters, but we could pull `Unix.stat` into `ocamltest_unix` (as the machinery is there for other reasons) and reject directories containing symlinks?